proposals/lip-0046.md

```diff
@@ -225,11 +230,11 @@ This function is used to return the random seed as a 16-bytes value.
 It is specified as:
 
 ```python
-getRandomBytes(height, numberOfSeeds):
+def getRandomBytes(height: uint32, numberOfSeeds: int) -> bytes:
```

Comment:
Change the type of `numberOfSeeds` to `uint32`, we decided to be more specific with integer types.